src/components/atoms/NotificationBar/NotificationBar.tsx

```diff
@@ -0,0 +1,21 @@
+import * as classNames from 'classnames'
+import * as React from 'react'
+import './NotificationBar.scss'
+
+interface NotificationBarProps {
```

Comment:
This is very similar to NotificationBar from src/interfaces/Props.ts.

Accidental duplication?

Reply:
Got it. So redux state interfaces are in interfaces/Props, but Props interfaces are in the same file as the components?

Reply:
yes